annotation-processing/src/test/java/com/exactpro/epfast/annotation/processing/FastProcessorTest.java

```diff
@@ -15,7 +16,11 @@ void testProcessing() {
             .compilesWithoutError()
             .and().generatesFileNamed(StandardLocation.CLASS_OUTPUT,
             "com.exactpro.epfast.annotation.internal",
-            "CreatorImpl.class");
+            "CreatorImpl.class")
+            .and().generatesFileNamed(StandardLocation.CLASS_OUTPUT,
+            "META-INF.services",
```

Comment:
It seems it works, but it's not valid package name. So I'd rather move it to next parameter as META-INF/services/